pandapower/opf/PpPmInterface/test/runtests.jl

```diff
@@ -1,6 +1,28 @@
+using PowerModels
```

Comment:
all depending and models are defined inside the PpPmInterface pkg, no need to call them again.